app/models/spree/gateway/mollie_gateway.rb

```diff
@@ -164,7 +190,7 @@ def update_by_mollie_status!(mollie_payment, payment)
           payment.complete! unless payment.completed?
           payment.order.finalize!
           payment.order.update_attributes(:state => 'complete', :completed_at => Time.now)
-        when 'cancelled', 'expired', 'failed'
+        when 'canceled', 'expired', 'failed'
           payment.failure! unless payment.failed?
         when 'refunded'
```

Comment:
`refunded` status has been removed in `v2`.

Reply:
Removed it ✅